### PR TITLE
Prepare for initial release

### PR DESCRIPTION
**Pull request by carllerche**

### CODE REVIEW

File: tower-grpc-build/Cargo.toml [hunk 1]
```diff
@@ -2,8 +2,15 @@
 name = "tower-grpc-build"
 version = "0.1.0"
 edition = "2018"
-authors = ["Carl Lerche <me@carllerche.com>"]
+authors = ["Tower Maintainers <team@tower-rs.com>"]
 license = "MIT"
+readme = "README.md"
+repository = "https://github.com/tower-rs/tower-grpc"
+homepage = "https://github.com/tower-rs/tower-grpc"
+documentation = "https://docs.rs/tower-grpc/0.1.0"
```
Comment (LucioFranco) [inline tower-grpc-build/Cargo.toml]: tower-grpc-build

Comment (carllerche) [reply tower-grpc-build/Cargo.toml]: thanks 👍

File: tower-grpc/src/lib.rs [hunk 2]
```diff
@@ -1,4 +1,10 @@
-#![deny(warnings, missing_debug_implementations, rust_2018_idioms)]
+#![doc(html_root_url = "https://docs.rs/tower-grpc/0.1.0")]
+#![deny(missing_debug_implementations, rust_2018_idioms)]
+// TODO: enable when there actually are docs
+// #![deny(missing_docs)]
```
Comment (LucioFranco) [inline tower-grpc/src/lib.rs]: TODO?

Comment (carllerche) [reply tower-grpc/src/lib.rs]: Many types don’t have docs. Also, I expect the vast majority will go away in 0.2 when we switch to async / await. I rather not spend time on it now. 

This release is mostly about just putting what we have on crates.io

Comment (LucioFranco) [reply tower-grpc/src/lib.rs]: fine with me!

**Comment by LucioFranco** [comment]
:shipit: 🔥 🔥 🔥 🚀

**Pull request merged**